author.rb

```diff
@@ -0,0 +1,14 @@
+class Author
+  attr_accessor :items
+
+  def initialize(id, first_name, last_name)
+    @id = id
+    @first_name = first_name
+    @last_name = last_name
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
```

Comment:
You have to include this line `item.add_author(self)` after line 12 to meet the requirement below as stated in the project.
`- should add self as a property of the item object (by using the correct setter from the item object)`